website_backend_views/model/ir_qweb.py

```diff
@@ -24,7 +24,7 @@
 
 
 class IrQweb(models.Model):
```

Comment:
@yvaucher I was tempted to quote Shakespeare on this one, but you're right of course. As a followup, the file name and the import have to change too